airlock/templates/file_browser/contents.html

```diff
@@ -1,8 +1,46 @@
 <div id="selected-contents">
 
-  {% if path_item.is_directory %}
+  {% if path_item.type.name == "FILEGROUP" %}
 
-    {% #card title=path_item.name %}
+    {% #card title=path_item.name container=True %}
+      <form action="{{ group_edit_url }}" method="POST" aria-label="group-edit-form">
+        {% csrf_token %}
+        <div style="display: flex; flex-direction: row; width: 100%; flex-wrap: wrap; gap: 1rem;">
+          <div style="display: flex: flex-direction: column; flex-basis: 100%; flex: 1;">
+            {% form_textarea field=group_edit_form.context show_placeholder=True resize=True placeholder="Describe the data to be released in this file" class="w-full max-w-lg mx-auto" rows=6 %}
```

Comment:
"in this file" -> "in the files in this group"/"in these files"?

Reply:
"to this group of files"?